distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1009,10 +1009,7 @@ type Column
        - range: The selection of rows from the table to return.
     @range Index_Sub_Range.default_widget
     take : (Index_Sub_Range | Range | Integer) -> Column
-    take self range=(First 1) =
-        _ = range
-        msg = "`Column.take` is not yet implemented."
-        Error.throw (Unsupported_Database_Operation.Error msg)
+    take self range=(First 1) = self.to_table.take range . at self.name
```

Comment:
```suggestion
    take self range=(First 1) = self.to_table.take range . at 0
```

nitpick: IMHO it's better to select by index as it spares us from text-comparisons. Will not matter for performance, but IMO it looks simpler. But feel free to ignore as that works just as well (although in the past before #5951 there was some trouble with using by-name vs by-index selection in places like this).

Reply:
Done